src/core/Table/filters/FilterToggle.tsx

```diff
@@ -11,19 +11,21 @@ import '@itwin/itwinui-css/css/table.css';
 import { useTheme } from '../../utils/hooks/useTheme';
 import { Popover } from '../../utils/Popover';
 import { getDocument } from '../../utils/common';
+import { IconButton } from '../../Buttons';
+import { StylingProps } from '../../utils/props';
 
 export type FilterToggleProps<T extends Record<string, unknown>> = {
   column: HeaderGroup<T>;
   ownerDocument?: Document;
-};
+} & StylingProps;
```

Comment:
Is this needed as we don't expose `FilterToggle` in `index.ts`?

Reply:
I added it just in case there is some need for it (internally or externally).

My first use case was to add padding-right, but I think I will do that in css